src/components/Skeleton/Skeleton.module.css

```diff
@@ -4,24 +4,24 @@
 
 @keyframes bg-wipe {
   0% {
-    background-position: 0% 0%;
+    background-position: right 0%;
   }
   100% {
-    background-position: 100% 0%;
+    background-position: left 0%;
```

Comment:
What's the benefit of this change?

Reply:
Good question. I use a linear gradient for the background image, which is always 5x the width of the container.  the way `background-position` works is that it will position the background within the container.

since the bg is bigger, we want it to start as pinned to the **right** edge (and go off screen on the left side), then transition to pinned to **left** edge (and go off screen to the right side). It's a little counter-intuitive b/c of the larger bg size (hence the bug lol).

https://developer.mozilla.org/en-US/docs/Web/CSS/background-position#regarding_percentages